engine/src/main/java/org/hibernate/validator/internal/engine/ConstraintViolationImpl.java

```diff
@@ -213,6 +227,11 @@ public Object getExecutableReturnValue() {
 	}
 
 	@Override
+	public Object getViolationContext() {
+		return violationContext;
+	}
+
+	@Override
 	// IMPORTANT - some behaviour of Validator depends on the correct implementation of this equals method! (HF)
```

Comment:
IMHO ```violationContext``` should *not* be taken into account for the equals/hashcode methods the same reason as ```expressionVariables```. I did not document this... should we add that to the doc?